frontend-park-mail-ru/2024_2_ThereWilIBeName
# rk2 (#22)

* home page fix

* home page fix test

* home page fix test 2

* src fix

* profile page in progress

* profile in progress 2

* profile in progress 3

* complete profile page & complete trips page

* fix margin menu & fix static urls

* fix static dev server

* fix router

* unused default classes for user & places

* place page added

* backup fix 1

* fix trips.styl

* fix stylus files

* fix side menu stylus

* fix design

* timur merge

* timur merge 2

* timur merge 3

* timur merge 4

* timur merge 5

* add getUserTrips

* add create-trip page

* fix logout

* fix getTrips

* fix getTrips gallery

* fix createTrip date

* Place fix (#15)

* fixed

* urls and placelistener fixed

* paths

* fixed data

* fix getUserTrips

* fix deleteTrip css + place css

* add deleteTrip

* add deleteTrip

* fix timur govno

* registration fix

* delete trip fix

* fix

* again fixes

* fix timur govno 2

* fix timur govno 3

* fix place side menu

* fix place side menu 2

* create sid-menu module

* create sid-menu module

* fix side-menu module

* add side menu to trips

* fix trips side menu

* fix trips side menu 2

* fix trips unauthorized

* fix trips unauthorized 2

* test fix mixin

* fix mixinBlock

* fix mixinBlock 2

* fix mixinBlock 3

* fix mixinBlock 4

* fix mixinBlock 5

* fix mixinBlock 6

* fix mixinBlock 7

* fix mixinBlock 8

* fix mixinBlock 9

* fix mixinBlock 10

* add formatDate

* edit delete.png

* fix delete review button

* fix delete review button

* fix delete review button 2

* fix trip delete button

* fix place.styl

* delete review button hover add

* delete review button margin add

* delete review button margin fix

* trip title fix

* fix trip item

* test trip item column-gap

* fix timur &

* fix profile gallery text

* fix profile gallery text

* fix profile gallery text 2

* fix profile gallery text 3

* fix profile gallery text 4

* fix profile gallery text 5

* fix profile gallery text 5

* fix profile gallery text 6

* old version

* profile edit button input add

* map.png add

* edit profile button fix

* fix map + profile input add

* fix map 2

* fix map 3

* fix map 4

* edit edit-profile-input

* add confirm.png

* fix confirm.png

* fix padding confirm.png

* add change password button

* add 232323.png

* remove 232323.png

* fix trips

* fix trips 2

* fix trips 3

* fix trips 4

* add 232323.png in new way

* add 232323.png in new way 2

* add in progress png

* fix in progress png

* fix in progress png 2

* add getUserReviews

* fix date function

* fix html width

* fix html

* add service-worker

* add service-worker

* add favicon

* add change-password

* add putUserInformation & putChangePassword

* add myImg

* fix myImg

* add avatar

* fix reviews

* add edit trip

* add please block

* fix please block

* fix please block

* delete reviews fixed

* micro fixes

* in progress palms

* in progress palms fix

* in progress palms fix 2

* fix routing

* delete headerMount

* delete headerMount

* fix profile

* api errors fix

* api errors fix 2

* fix change-password

---------
